apps/andi/test/integration/andi_web/live/data_dictionary_field_editor_test.exs

```diff
@@ -158,7 +159,7 @@ defmodule AndiWeb.EditLiveView.DataDictionaryFieldEditorTest do
 
   describe "certain fields in the data dictionary editor are unavaliable for non curator users" do
     setup %{curator_subject: curator_subject, public_subject: public_subject} do
-      {:ok, public_user} = Andi.Schemas.User.create_or_update(public_subject, %{email: "bob@example.com"})
+      {:ok, public_user} = Andi.Schemas.User.create_or_update(public_subject, %{email: "bob162@example.com"})
```

Comment:
Oops - was using these to track down a leaky database sandbox case. I can revert if desired.